dashboard/db/migrate/20200803211022_update_academic_year_workshop_subjects.rb

```diff
@@ -47,6 +47,7 @@ def up
       csd_workshops.each do |csd_workshop|
         csd_workshop.update(
           subject: subject_name[:to],
+          virtual: true,
```

Comment:
this will always set to virtual right? I think we should only do that if it's one of virtual 1/3/5/7

Reply:
🤦 yes, thanks!